output/src/main/java/com/scottlogic/deg/output/guice/OutputConfigSource.java

```diff
@@ -21,6 +21,8 @@
 public interface OutputConfigSource {
     OutputFormat getOutputFormat();
     Path getOutputPath();
+    /** this is currently only used if outputFormat is SQL */
```

Comment:
This seems like a violation of the interface segregation principle? However, I'm not sure if there's a neater way to insert it in the area without notably more effort.

Reply:
yeah, its not ideal that its added purely for sql outputter